code/src/test/resources/testFirstIndexWhereStdlib.hcl

```diff
@@ -1,3 +1,3 @@
 # This should return 3
-
+# firstIndexWhere is NOT found in std.hcl?
```

Comment:
this is my mistake, wrote it before i found stdlib.hcl
(i was pissed at the time srry)

Reply:
Fixed